ui/app/src/state/actions/preview.ts

```diff
@@ -207,11 +207,7 @@ export const checkInGuest = /*#__PURE__*/ createAction<{
   documentDomain?: string;
 }>(guestCheckIn.type);
 
-export function checkOutGuest(): StandardAction {
-  return {
-    type: guestCheckOut.type
-  };
-}
+export const checkOutGuest = createAction(guestCheckOut.type);
```

Comment:
This and the one just above it look like they're just duplicates of `guestCheckOut`and `guestCheckIn`. We should remove them.